src/bundle/Resources/views/content/tabs/relations/table.html.twig

```diff
@@ -0,0 +1,40 @@
+{% trans_default_domain "locationview" %}
+
+{% if relationList %}
+    <table class="ez-table-data">
+        <thead>
+            <tr>
+                <th>{{ 'locationview.relations.name'|trans()|desc('Name') }}</th>
+                <th>{{ 'locationview.relations.content.type'|trans()|desc('Content type') }}</th>
+                <th>{{ 'locationview.relations.relation.type'|trans()|desc('Relation type') }}</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for relation in relationList %}
+            <tr>
+                <td><a href="{{ path(relation.destinationLocation) }}">{{ relation.destinationContentInfo.name }}</a></td>
+                <td>{{ relation.destinationContentTypeName }}</td>
+                <td>
+                    {% if relation.type == constant('eZ\\Publish\\API\\Repository\\Values\\Content\\Relation::COMMON') %}
```

Comment:
Is this comment on the right line, @dpobel ? I don't see how it related to the `constant` usage.

Or were you talking about a `type` property, that would contain the translation label ?